### PR TITLE
Add doc for cylinder3d optimized version

### PR DESCRIPTION
**Pull request by Liu-xiandong**
Add doc for cylinder3d optimized version

### CODE REVIEW

File: examples/laplace/README.md [hunk 1]
```diff
@@ -136,3 +136,12 @@ rslt = solution(geo).numpy()
 psci.visu.save_vtk(geo, rslt, 'rslt_laplace_2d')
 np.save(rslt, 'rslt_laplace_2d.npy')
 ```
+
+### Scalability on Distributed Systems
+Distributed is currently supported. The scalability performance is as follows:
+|gpus | points |  performance (sec/epoch) |
```
Comment (xingfeng01) [inline examples/laplace/README.md:142]: Number of GPU | Number of points | Performance (sec/epoch)
讲的太简单了

File: examples/laplace/README.md [hunk 2]
```diff
@@ -136,3 +136,12 @@ rslt = solution(geo).numpy()
 psci.visu.save_vtk(geo, rslt, 'rslt_laplace_2d')
 np.save(rslt, 'rslt_laplace_2d.npy')
 ```
+
+### Scalability on Distributed Systems
+Distributed is currently supported. The scalability performance is as follows:
+|gpus | points |  performance (sec/epoch) | 
+|---|---|---|
+|N1C1 | 580w * 1 | 0.9483 s|
```
Comment (xingfeng01) [inline examples/laplace/README.md:144]: N*C* 缩略看不懂

Comment (xingfeng01) [reply examples/laplace/README.md:144]: w 这个单位英文不存在

File: examples/laplace/README.md [hunk 3]
```diff
@@ -136,3 +136,12 @@ rslt = solution(geo).numpy()
 psci.visu.save_vtk(geo, rslt, 'rslt_laplace_2d')
 np.save(rslt, 'rslt_laplace_2d.npy')
 ```
+
+### Scalability on Distributed Systems
+Distributed is currently supported. The scalability performance is as follows:
```
Comment (xingfeng01) [inline examples/laplace/README.md:141]: Distributed computing

Comment (xingfeng01) [inline examples/laplace/README.md:141]: 加上怎么运行的命令吧

File: examples/cylinder/3d_unsteady_discrete/README.md [hunk 4]
```diff
@@ -4,206 +4,27 @@
...
-The physical information of the interior point at the current moment is added through `solver.feed_data_interior_cur`. 
-The physical information of the user point at the current moment is added through `solver.feed_data_user_cur`. 
-The physical information of the user point at the next moment is added through `solver.feed_data_user_next`.
-
-After all the data is ready, it can be trained with `solver.solve`. 
-Finally, the physical information of the next moment is solved.
-
+## Optimize
+If you want to try out the advanced automatic differentiation function of static graph, you can run the following command.
+Compared with the baseline version, it can get 15% performance improvement.
```
Comment (levi131) [inline examples/cylinder/3d_unsteady_discrete/README.md:21]: `it can get 15% performance improvement.` 建议修改为
`Due to the new automatic differentiation mechanism, calculation diagram optimization, new actuator execution optimization technology and distributed technology, there are improvements in speed and memory occupation, and can support multi machine and multi card training.`

Comment (levi131) [inline examples/cylinder/3d_unsteady_discrete/README.md:21]: 建议把这一行删除。性能优势放在单独的readme中讲。

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[//]: <> (title: Flow around a cylinder use case tutorial, author: Xiandong Liu @liuxiandong at baidu.com)
+
+
+# Flow around a Cylinder
+
+This guide introduces to how to build a PINN model for simulating the flow around a cylinder in PaddleScience.
+
+This demo is based on Paddle static graph mode, including the following parts:
+
+ - Use v1.0 API, including geometry, pde, bc, network, algorithm and other parts;
```
Comment (levi131) [inline examples/cylinder/3d_unsteady_discrete/optimize/README.md:10]: Use the 1.0 API of PaddleScience suite to complete the construction of geometry, PDE, BC, network, algorithm, etc

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+ - Note that the loss part is directly implemented based on the new automatic differentiation mechanism of the Paddle framework, and the provided procedural automatic differentiation interface, completes the definition of the eq_loss part, and forms the overall loss together with bc_loss and data_loss;
+ 
+ - Distributed data parallelism + gradient accumulation technology, computational graph optimization technology, and new executor execution optimization technology are used in the demo.
```
Comment (levi131) [inline examples/cylinder/3d_unsteady_discrete/optimize/README.md:14]: After the program is built, the distributed data parallel and gradient accumulation technology, computational graph optimization technology and new actuator execution optimization technology are used

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+In terms of `performance`, due to the new automatic differentiation mechanism, calculation graph optimization, and new executor execution optimization technology, the performance and video memory have been greatly improved. It is worth mentioning that due to the improvement of the new automatic differentiation mechanism in computing logic 15% increase.
```
Comment (levi131) [inline examples/cylinder/3d_unsteady_discrete/optimize/README.md:16]: there are significant improvements in performance and memory occupation. It is worth mentioning that due to the improvement of calculation logic of the new automatic differentiation mechanism, the single machine performance is improved by 15%.

File: examples/laplace/README.md (hunk 1 above)
Comment (Liu-xiandong) [reply examples/laplace/README.md]: Done.

File: examples/laplace/README.md (hunk 2 above)
Comment (Liu-xiandong) [reply examples/laplace/README.md]: Done.

File: examples/laplace/README.md (hunk 3 above)
Comment (Liu-xiandong) [reply examples/laplace/README.md]: Done.

Comment (Liu-xiandong) [reply examples/laplace/README.md]: Done.

File: examples/cylinder/3d_unsteady_discrete/README.md (hunk 4 above)
Comment (Liu-xiandong) [reply examples/cylinder/3d_unsteady_discrete/README.md]: Done.

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md (hunk 5 above)
Comment (Liu-xiandong) [reply examples/cylinder/3d_unsteady_discrete/optimize/README.md]: Done.

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md (hunk 6 above)
Comment (Liu-xiandong) [reply examples/cylinder/3d_unsteady_discrete/optimize/README.md]: Done.

File: examples/cylinder/3d_unsteady_discrete/optimize/README.md (hunk 7 above)
Comment (Liu-xiandong) [reply examples/cylinder/3d_unsteady_discrete/optimize/README.md]: Done.

**Review by xingfeng01 — APPROVED** [review]

**Review by levi131 — APPROVED** [review]
LGTM
